src/masternode/masternode-meta.h

```diff
@@ -73,7 +73,7 @@ class CMasternodeMetaInfo
 
     bool IsValidForMixingTxes() const { return GetMixingTxCount() <= MASTERNODE_MAX_MIXING_TXES; }
 
-    // KEEP TRACK OF EACH GOVERNANCE ITEM INCASE THIS NODE GOES OFFLINE, SO WE CAN RECALC THEIR STATUS
+    // KEEP TRACK OF EACH GOVERNANCE ITEM IN CASE THIS NODE GOES OFFLINE, SO WE CAN RECALCULATE THEIR STATUS
```

Comment:
Any reason to keep this comment so aggressive with uppercase? 😄

Reply:
Agree but out of scope